src/exchange_adapters/coinbase.ts

```diff
@@ -62,13 +62,18 @@ export class CoinbaseAdapter extends BaseExchangeAdapter implements ExchangeAdap
    */
   parseTicker(json: any): Ticker {
     const lastPrice = this.safeBigNumberParse(json.price)!
+    const baseVolume = this.safeBigNumberParse(json.volume)!
+    // Using lastPrice to convert from baseVolume to quoteVolume, as CoinBase's
+    // API does not provide this information.
```

Comment:
In your other PR, you added some context about why to use `lastPrice` rather than the midpoint between ask + bid. Can you add that explanation in this comment?

Reply:
Done.